src/app/shared/components/add-external-cluster-dialog/steps/provider/template.html

```diff
@@ -27,8 +27,7 @@ <h3>Import</h3>
     <h3 fxLayoutGap="10px"
         fxLayoutAlign=" center">
       <span>Connect</span>
-      <i class="km-icon-info"></i>
-      <span class="tooltip">Connected clusters can only be viewed, not edited</span>
+      <mat-hint class="tooltip">Connected clusters can only be viewed, not edited</mat-hint>
```

Comment:
This needs to be updated like this:

https://user-images.githubusercontent.com/17727069/175227010-d133ce57-38d4-4fe7-ae08-f7b778accf3b.mp4

Reply:
@ahmedwaleedmalik specifically told me not to do this

Reply:
@Waseem826  thoughts?

Reply:
After discussion with @cschieder we have decided upon this design 
<img width="676" alt="image" src="https://user-images.githubusercontent.com/56511988/175930300-44d94620-9379-4cb4-8685-cdef73b66ff3.png">